features/update/activities/feature.ts

```diff
@@ -0,0 +1,38 @@
+import { Feature } from '@temporalio/harness';
+import * as wf from '@temporalio/workflow';
+import * as assert from 'assert';
+
+const myUpdate = wf.defineUpdate<void, any[]>('myUpdate');
```

Comment:
`defineSignal` and `defineQuery` have defaults for their `Args` types params of an empty array so you don't have to specify input types if there are none, should we do the same here? I wonder if we should even have a default of `void` for the `Ret` type since parameterless void updates may be normal.

Reply:
I am ok requiring an update to specify its return type for two reasons: 1) we can come back later and add a default if we want, and 2) it makes it clear that updates can return if they want. However, I am also ok defaulting it to void for two reasons: 1) people will default to `any` when they don't care, and 2) it will break their TS compilation if they accidentally add a non-void method as a handler without having explicitly specified the expected return type.

So I have no strong opinion here. Would like to hear other team members' opinions, especially those that work w/ this SDK most.